Configurations/unix-Makefile.tmpl

```diff
@@ -587,17 +596,27 @@ install_docs: install_man_docs install_html_docs
 uninstall_docs: uninstall_man_docs uninstall_html_docs
 	$(RM) -r $(DESTDIR)$(DOCDIR)
 
-install_fips: install_sw
+install_fips:
+	@[ -n "$(INSTALLTOP)" ] || (echo INSTALLTOP should not be empty; exit 1)
+	@$(PERL) $(SRCDIR)/util/mkdir-p.pl $(DESTDIR)$(MODULESDIR)
+	@$(ECHO) "*** Installing FIPS module"
+	@$(ECHO) "install $(INSTALL_FIPS) -> $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME)"
+	@cp "$(INSTALL_FIPS)" $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME).new
+	@chmod 755 $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME).new
+	@mv -f $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME).new \
+	       $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME)
 	@$(ECHO) "*** Installing FIPS module configuration"
-	@$(ECHO) "fipsinstall $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME).cnf"
-	@openssl fipsinstall -module $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME) \
-		-out $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME).cnf \
-		-macopt 'key:$(FIPSKEY)'
+	@$(ECHO) "fipsinstall $(OPENSSLDIR)/fipsmodule.cnf"
+	@$(BLDDIR)/util/wrap.pl $(BLDDIR)/apps/openssl fipsinstall \
+		-module $(DESTDIR)$(MODULESDIR)/$(FIPSMODULENAME) \
+		-out $(OPENSSLDIR)/fipsmodule.cnf \
```

Comment:
Should this be rather `$(DESTDIR)$(OPENSSLDIR)/fipsmodule.cnf`

Reply:
Thanks, you are right.

Reply:
Fixed in def19f6dd37c7e4c973c664926665129fc673af5 (all locations).